content/en/docs/setup/install/multicluster/primary-remote/snips.sh

```diff
@@ -58,6 +58,10 @@ kubectl apply --context="${CTX_CLUSTER1}" -n istio-system -f \
     samples/multicluster/expose-istiod.yaml
 }
 
+snip_expose_the_control_plane_in_cluster1_2() {
```

Comment:
@xu-jianchu Since we're not actually testing these new snips, it would be good if you would at least manually test them by copy/pasting them from the preview to make sure they are working. https://deploy-preview-14571--preliminary-istio.netlify.app/latest/docs/setup/install/multicluster/primary-remote/#expose-the-control-plane-in-cluster1

Reply:
Thank you @frankbu! I only tested the commands locally but didn't realize there's a preview with everything generated. Will do!